README.md

```diff
@@ -0,0 +1 @@
+# robos
```

Comment:
Empty readme, be sure to add a readme in the future ;)

Reply:
we use the wiki for that, right? :)